src/test/java/domain/user/PlayerTest.java

```diff
@@ -0,0 +1,173 @@
+        );
+    }
+
+    @DisplayName("플레이어의 카드 추가 기능을 테스트한다.")
+    @Test
+    void addCardTest() {
+        // given
+        PlayerName playerName = new PlayerName("player");
+        BettingMoney bettingMoney = new BettingMoney(new BigDecimal(1000));
+        Player player = new Player(playerName, bettingMoney);
+
+        // when
+        player.addCard(new Card(Symbol.DIAMOND, Type.ACE));
+
+        // then
+        assertThat(player.getCards()).hasSize(1);
+    }
+
+    @DisplayName("플레이어의 카드 뽑기 기능을 테스트한다.")
+    @Test
+    void hitTest() {
+        // given
+        Deck deck = new Deck();
+        PlayerName playerName = new PlayerName("player");
+        BettingMoney bettingMoney = new BettingMoney(new BigDecimal(1000));
+        Player player = new Player(playerName, bettingMoney);
+
+        // when
+        player.hit(deck);
+
+        // then
+        assertThat(player.getCards()).hasSize(1);
+    }
+
+    @DisplayName("플레이어가 자신의 턴을 수행하는 기능을 테스트한다.")
+    @ParameterizedTest
+    @MethodSource("provideIsStayAndCardsSize")
+    void proceedOwnTurnTest(boolean isStay, int cardsSize) {
+        // given
+        PlayerName playerName = new PlayerName("player");
+        BettingMoney bettingMoney = new BettingMoney(new BigDecimal(1000));
+        Player player = new Player(playerName, bettingMoney);
+        Deck deck = new Deck();
+
+        // when
+        player.proceedOwnTurn(isStay, deck);
+
+        // then
+        assertThat(player.getCards()).hasSize(cardsSize);
+    }
+
+    private static Stream<Arguments> provideIsStayAndCardsSize() {
+        return Stream.of(
+                Arguments.of(true, 0),
+                Arguments.of(false, 1)
+        );
+    }
+
+    @DisplayName("플레이어의 카드 조합이 블랙잭이 아닌 경우 카드 합 계산 기능을 테스트한다.")
+    @Test
+    void calculateTotalCardNumberWhenIsNotBlackJackTest() {
+        // given
+        PlayerName playerName = new PlayerName("player");
+        BettingMoney bettingMoney = new BettingMoney(new BigDecimal(1000));
+        Player player = new Player(playerName, bettingMoney);
+        player.addCard(new Card(Symbol.SPADE, Type.KING));
+        player.addCard(new Card(Symbol.HEART, Type.KING));
+        player.addCard(new Card(Symbol.CLUB, Type.KING));
+
+        // when
+        int totalCardNumber = player.calculateTotalCardNumber();
+
+        // then
+        assertThat(totalCardNumber).isEqualTo(30);
+    }
+
+    @DisplayName("플레이어의 카드 조합이 블랙잭인 경우 카드 합 계산 기능을 테스트한다.")
+    @Test
+    void calculateTotalCardNumberWhenIsBlackJackTest() {
+        // given
+        PlayerName playerName = new PlayerName("player");
+        BettingMoney bettingMoney = new BettingMoney(new BigDecimal(1000));
+        Player player = new Player(playerName, bettingMoney);
+        player.addCard(new Card(Symbol.SPADE, Type.ACE));
+        player.addCard(new Card(Symbol.SPADE, Type.KING));
+
+        // when
+
+        // then
+        assertThat(player.calculateTotalCardNumber()).isEqualTo(21);
```

Comment:
개인적인 의견이지만, 아래와 같이 when, then의 구조도 정확하게 지키면 좋을 것 같아요! (given, when, then 구조를 사용하실 거라면!)

```suggestion
        // when
        int actual = player.calculateTotalCardNumber();

        // then
        assertThat(actual).isEqualTo(21);
```